src/Pact/Types/Command.hs

```diff
@@ -128,7 +129,7 @@ mkCommand' creds env = Command env (sig <$> creds) hsh
 verifyCommand :: FromJSON m => Command ByteString -> ProcessedCommand m ParsedCode
 verifyCommand orig@Command{..} = case (ppcmdPayload', ppcmdHash', mSigIssue) of
       (Right env', Right _, Nothing) -> ProcSucc $ orig { _cmdPayload = env' }
-      (e, h, s) -> ProcFail $ "Invalid command: " ++ toErrStr e ++ toErrStr h ++ fromMaybe "" s
+      (e, h, s) -> ProcFail $ CommandError "Invalid command" $ Just $ toErrStr e ++ toErrStr h ++ fromMaybe "" s
```

Comment:
I missed one small detail here - there's an extra `($)`

Reply:
I think all of them are necessary (or would need to be replaced with (.) or parentheses). Which one   do you mean?

Reply:
I misread, sorry. You're fine.